### ISSUE TITLE
Composer Error at 1.1.24

### ISSUE THREAD

**Issue by romanzipp** [planetteamspeak/ts3phpframework#8]
Please update `composer.json` with the correct composer package name "`ts3-php-framework`" [see here](https://packagist.org/packages/planetteamspeak/ts3-php-framework).

The current name, written in the composer file is
```json
{
  "name": "planetteamspeak/ts3phpframework"
}
```

**Comment by ronindesign** [comment]
+1 I can confirm this.

When using composer command:

`$ composer require planetteamspeak/ts3phpframework`

Result:
```
[InvalidArgumentException]                                                                                                           
  Could not find package planetteamspeak/ts3phpframework at any version for your minimum stability (stable). Check the package spelling or your minimum-stability
```

Either `composer.json` or Packagist entry need to be updated. From what I've seen, Packagist packages typically are named the same as their git hub repo name, so maybe changing the Packagist name to `planetteamspeak/ts3phpframework` would be more consistent.

**Comment by svenpaulsen** [comment]
Thanks for reporting this. I've merged ronindesign's pull request #9 so this should be fixed now.
